docker/docker-compose.test.yml

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright 2016-2019 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+version: "3.6"
+volumes:
+  gcp-service-account:
+services:
+  extract-service-account:
+    image: busybox
+    volumes:
+      - gcp-service-account:/credentials:rw
+    command: sh -c 'echo $GOOGLE_APPLICATION_CREDENTIALS_BASE64 | base64 -d > /credentials/service-account-key.json'
+    environment:
+      - GOOGLE_APPLICATION_CREDENTIALS_BASE64
+  show-service-account:
+    image: busybox
+    volumes:
+      - gcp-service-account:/credentials:ro
+    command: sh -c 'ls -lh /credentials/service-account-key.json'
```

Comment:
I almost thought we were leaking the credential ;) Probably can add `# Show that the file exists as a sanity check in logs.`

Reply:
:) yeah, i noticed the cat and changed it immediately :p

Reply:
updated w/ more comment.